ci/run-tests.sh

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env bash
```

Comment:
I'd rather keep the prompt considering that this script is highly bash-specific.